File: 0268-missing-number/0268-missing-number.c
int missingNumber(int* nums, int n) {
    int count=0;
    int sum=0;
    for(int i=0;i<=n;i++){
        count=count+i;
    }
  for(int j=0;j<n;j++){
    sum=sum+nums[j];
  }
    return count-sum;
}

File: 0268-missing-number/0268-missing-number.java
class Solution {
    public int missingNumber(int[] nums) {
        int n=nums.length;
        Arrays.sort(nums);
        int miss=n;
        for(int i=0;i<n;i++)
        {
            if(i!=nums[i]){
                miss=i;
                break;
            }
        }
        return miss;
        
        
    }
}